binding.cc

```diff
@@ -1179,6 +1180,26 @@ namespace zmq {
     NanReturnValue(NanNew<String>(version_info));
   }
 
+#if ZMQ_VERSION_MAJOR >= 4
+   static NAN_METHOD(ZmqCurveKeypair) {
+    NanScope();
+
+    char public_key [41];
+    char secret_key [41];
+
+    int rc = zmq_curve_keypair( public_key, secret_key);
+    if (rc < 0) {
+      NanThrowError("zmq_curve_keypair operation failed. Method support in libzmq v4+ -with-libsodium.");
+    }
```

Comment:
so one more thing, @szilardhuber before you push again you need to either:
```c
return NanThrowError("zmq_curve, etc."); 
``` 
or if that doesnt compile 
```c
NanThrowError("zmq_curve, etc.");
return; 
``` 
so that we stop execution immediately before `Local<Object> obj`